ocs_ci/utility/utils.py

```diff
@@ -1188,3 +1188,21 @@ def get_rook_repo(branch='master', to_checkout=None):
     run_cmd(f"git reset --hard origin/{branch}", cwd=cwd)
     if to_checkout:
         run_cmd(f"git checkout {to_checkout}", cwd=cwd)
+
+
+def get_latest_ds_olm_tag():
+    """
+    This function returns latest tag of OCS DS registry
```

Comment:
s/DS/Downstream/ and everywhere else for clarity.

Reply:
only in docstrings?  I think that the name of function can remain with ds for shorten version?

Reply:
yeah the docstring change should do + if using in comments we can use full name.

Reply:
done